README.md

```diff
@@ -13,14 +13,16 @@ The purpose of this package is to provide an easily configurable OAuth 2 server,
 
 ## Development prerequisites
 
-- [Node.js 8.0+](https://nodejs.org/)
+- [Node.js 10.0+](https://nodejs.org/)
+- [Yarn 1.15.2+](https://classic.yarnpkg.com/lang/en/)
 
 ## How to use
 
 Add it to your Node.js project as a development dependency:
 
 ```shell
 npm install --save-dev oauth2-mock-server
+yarn add -D oauth2-mock-server
```

Comment:
This should be like `npm` OR `yarn`. The way it's written, it seems that you need to run both.

Reply:
@poveden Fixed